src/Cli/Pytocs.Cli.csproj

```diff
@@ -1,14 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net471;netcoreapp2.1</TargetFrameworks>
     <OutputType>exe</OutputType>
-    <AssemblyName>pytocs</AssemblyName>
+    <AssemblyName>Pytocs.Cli</AssemblyName>
     <RootNamespace>Pytocs.Cli</RootNamespace>
```

Comment:
Won't this cause the name of the cli executable to be changed? I prefer typing `pytocs` at the command line rather than `Pytocs.Cli`.

Reply:
will be fixed

Reply:
Yes, it will, but I think will better rename Pytocs.Cli.exe to pytocs in a build script. I will add renaming operation in appveyor.yaml

Reply:
But why not just let the build process create the right thing in the first place?

Reply:
Pytocs.dll and pytocs.exe has same assembly name, so it cannon be both in the solution. Added batch files which refers to original executable.